src/renderers/webgl/WebGLState.js

```diff
@@ -599,7 +599,6 @@ function WebGLState( gl, extensions, paramThreeToGL ) {
 
 			}
 
-			currentBlending = blending;
 			currentPremultipledAlpha = premultipliedAlpha;
```

Comment:
Well, it's clearer, that's for sure. I think it is the same logic. Perhaps employing a `switch` statement instead of the long `if-else` would also help.